File: trunk/quote/Projects/PluginModel/template/DefaultWireProperties.vb
Imports System.ComponentModel
Imports System.Reflection

Imports Model.Quote

Namespace Template

    Public Class DefaultWireProperties
        Inherits Template.WireProperties

        Public Sub New(ByVal quoteDetail As Model.Template.Detail)
            MyBase.New(quoteDetail)
        End Sub

    End Class
End Namespace

File: quote/Projects/PluginModel/template/DefaultWireProperties.cs
namespace Model.Template
{
    using System;
    using System.Linq;

    /// <summary>
    /// The default wire properties.
    /// </summary>
    public sealed class DefaultWireProperties : Template.WireProperties
    {
        /// <summary>
        /// Initializes a new instance of the <see cref="DefaultWireProperties" /> class.
        /// </summary>
        /// <param name="quoteDetail">A quote detail object.</param>
        public DefaultWireProperties(Model.Template.Detail quoteDetail) : base(quoteDetail)
        {
            // Base class
        }
    }
}